src/components/Avatar/Avatar .tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from "react";
+
+import styled from "styled-components";
+
+interface AvatarProps {
+  imageSrc: string | null;
+  alt: string;
+  letter?: string;
+}
```

Comment:
лучше сделать так
```suggestion
type AvatarProps = {
  src: string;
  alt: string;
} | {src: null; letter: string}
```
В таком случае в компоненте вы будете четко разделять два состояния, есть картинка - показать картинку и alt, нет - показать letter, для того чтобы `ts` нормально работал, деструктуризировать не надо, просто пишите 
if (props.src) {
   return ...;
} 

return <EmptyAvatar>{props.letter}</EmptyAvatar>